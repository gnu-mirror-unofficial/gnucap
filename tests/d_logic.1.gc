verilog

// does not work yet. logic "model" is inaccessible from verilog
// paramset must support COMPONENTS...
paramset mos logic;
   .delay= 1n;
   .rise= 1n;
   .fall= 1n;
   .rs= 100.;
   .rw= 1.G ;
   .thh= 0.75;
   .thl= 0.25;
   .mr= 5.;
   .mf= 5.;
   .over=10k;
	.vmax= 5.;
   .vmin= 0.;
endparamset

list
end
