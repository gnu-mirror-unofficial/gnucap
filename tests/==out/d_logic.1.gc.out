paramset mos logic; 
             ^ ? paramset: base has incorrect type
.delay= 1n; 
^ ? .delay: no match
.rise= 1n; 
^ ? .rise: no match
.fall= 1n; 
^ ? .fall: no match
.rs= 100.; 
^ ? .rs: no match
.rw= 1.G ; 
^ ? .rw: no match
.thh= 0.75; 
^ ? .thh: no match
.thl= 0.25; 
^ ? .thl: no match
.mr= 5.; 
^ ? .mr: no match
.mf= 5.; 
^ ? .mf: no match
.over=10k; 
^ ? .over: no match
.vmax= 5.; 
^ ? .vmax: no match
.vmin= 0.; 
^ ? .vmin: no match
endparamset 
^ ? endparamset: no match
// does not work yet. logic "model" is inaccessible from verilog 
// paramset must support COMPONENTS... 
